src/mongo/db/pipeline/TDigest.cpp

```diff
@@ -106,7 +108,7 @@ TDigest::TDigest(
 // possible.  This implementation puts all additional memory in the
 // heap, so that if called from fiber context we do not smash the
 // stack.  Otherwise it is very similar to boost::spreadsort.
-TDigest TDigest::merge(Range<const double*> unsortedValues) const {
```

Comment:
I would say we get rid of this method completely.
We won't do the sorting in this class. It will always happen outside.